display.js

```diff
@@ -3,35 +3,59 @@ import UsersList from './users-list.js'
 import Heading from './heading.js'
 import Search from './search.js'
 import { instruments } from './users-list.js'
+import MusicianDetails from './musician-details.js'
 
 export default class Display extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
       hasSearched: false,
-      selectedInstrument: ''
+      selectedInstrument: '',
+      selectedMusician: null,
+      users: []
     }
     this.handleSearch = this.handleSearch.bind(this)
     this.handleReset = this.handleReset.bind(this)
     this.instrumentFilter = this.instrumentFilter.bind(this)
+    this.getUsers = this.getUsers.bind(this)
+    this.selectMusician = this.selectMusician.bind(this)
+    this.closeDetails = this.closeDetails.bind(this)
   }
   handleReset() {
     this.setState({ hasSearched: false })
     this.setState({ selectedInstrument: '' })
   }
-
   handleSearch() {
     this.setState({ hasSearched: true })
   }
   instrumentFilter(event) {
     this.setState({ selectedInstrument: event.target.textContent })
   }
+  selectMusician(event) {
+    const $id = event.target.id
+    const selectedMusician = this.state.users.filter(user => user.id == $id)
```

Comment:
Filter returns an `Array`, but if you only wanted _one_ object, it's better to use [`Array.prototype.find`]. That way the rest of your code doesn't need to keep indexing into an `Array` all over the place.